buenavista/http/schemas.py

```diff
@@ -57,11 +58,11 @@ class StatementStats(CamelModel):
 
 
 class QueryError(CamelModel):
-    message: Optional[str]
-    sql_state: Optional[str]
+    message: Optional[str] = None
+    sql_state: Optional[str] = None
```

Comment:
@jwills I can't know if this was intentional or not since you do have some `Optional[...]` fields with a `None` default value. I blindly added `None` as a default to all fields that had failing tests because of missing values but maybe that's not what you want.

Reply:
Nah it was just laziness— thank you!